e2e-tests/src/test/java/gov/nasa/jpl/aerie/e2e/types/ActivityType.java

```diff
@@ -5,14 +5,21 @@
 import java.util.Map;
 
 
-public record ActivityType(String name, Map<String, Parameter> parameters) {
+public record ActivityType(String name, Map<String, Parameter> parameters, ValueSchema computedAttributes) {
+  /**
+   * Create an activity pe with an empty computed attributes value schema.
```

Comment:
Minor, nonblocking: typo
```suggestion
   * Create an ActivityType with an empty computed attributes value schema.
```